### DISCUSSION TITLE
Automatize the process

### DISCUSSION THREAD

**Discussion by zmeanszachary** [FurkanGozukara/Stable-Diffusion#4] [Q&A]
Hello guys,
Thanks [FurkanGozukara](https://github.com/FurkanGozukara) for this magnificient code. Is there any posibility to automatize the process? I just need to swap 3 videos, can it do it witout my interruption?

**Comment by FurkanGozukara** [comment]
> Hello guys, Thanks [FurkanGozukara](https://github.com/FurkanGozukara) for this magnificient code. Is there any posibility to automatize the process? I just need to swap 3 videos, can it do it witout my interruption?

Could be possible but I didn't do that yet
